lms/db/_bulk_action.py

```diff
@@ -50,6 +52,9 @@ class Config:
         upsert_update_elements: List[str]
         """Columns to update when a match is found."""
 
+        upsert_trigger_onupdate: bool = True
+        """Column to update with the current datetime"""
```

Comment:
```suggestion
        """Update any columns with `onupsert` using the appropriate value."""
```

Also, this is probably a me brain thing, but `upsert`, `trigger`, and `onupdate` are all DB terms independently, which makes my brain read this as either:

* `upsert, trigger, onupdate`
* `(upsert trigger), onupdate`

Would something like `upsert_use_onupdate` be clearer?